cron/schedules.py

```diff
@@ -72,8 +72,9 @@ def download_schedule(api_key, username, year):
         req = requests.get(
             ENDPOINT_URL.format(username), headers=gen_auth_header(api_key), params={"term_id": str(term_id)}
         )
-        if req.status_code == 500:

```

Comment:
Let's put this file in its own PR since it will need some cleanup to figure out exactly what to do with the commented-out code.